ui/app/controllers/jobs/index.js

```diff
@@ -224,7 +225,13 @@ export default class IndexController extends Controller.extend(
     // A job must match ALL filter facets, but it can match ANY selection within a facet
     // Always return early to prevent unnecessary facet predicates.
     return this.visibleJobs.filter((job) => {
-      if (types.length && !types.includes(job.get('displayType'))) {
+      const shouldShowPack = types.includes('pack') && job.displayType.isPack;
+
+      if (types.length && shouldShowPack) {
+        return true;
+      }
+
+      if (types.length && !types.includes(job.get('displayType.type'))) {
```

Comment:
Something seems different about pack here; shown is a non-pack job where I have service + pack selected. Shouldn't this be hidden?


![image](https://github.com/hashicorp/nomad/assets/713991/71ee9df4-aed3-4307-926b-977a89a0750c)

Reply:
I guess my confusion is from Pack not really being a job type as much as an attribute of the job / making it the only non-mutually-exclusive item here. I don't think this will lead to much user error, so I think leave it in, but another way we might have tackled this is under a separate dropdown for job attributes.

Reply:
Yeah, it really does deserve its own filter. But then we just clutter the page. Our filtering is pretty far off from the rest of the other UIs. So it might be worth addressing later on down the line.